packages/mattermost-redux/src/actions/users_queries.ts

```diff
@@ -66,7 +73,7 @@ query MyData {
       scheme_user: schemeUser
       scheme_admin: schemeAdmin
     }
-}
+  }
 `;
 
 export const myDataQuery = JSON.stringify({query: myDataQueryString, operationName: 'MyData'});
```

Comment:
The operation name here is what needs to be changed :)

In any case, what I suggested was to rename `myData` to something more readable and intuitive. You have the word `myData` throughout this file - `myDataQueryString`, `MyDataQueryResponseType` etc. You'll have more and more gql queries, so we need names that correspond to what it serves in the UI. Maybe `gql-w-userInfo` is a better name? 0/5.

Reply:
oh forgot about the operation name only :P